lib/src/test/resources/compiled/providerTestAbi.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "inputs": [
+      {
+        "name": "amount",
+        "type": "felt"
+      }
+    ],
+    "name": "increase_balance",
+    "outputs": [],
+    "type": "function"
+  },
+  {
+    "inputs": [],
+    "name": "get_balance",
+    "outputs": [
+      {
+        "name": "res",
+        "type": "felt"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  }
+]
```

Comment:
Please add a flag to starknet-compile in compileContracts.sh to generate abi files

Reply:
Should we also add contents of `compiled/` to gitignore. Or do we want to commit compiled contracts and abis?

Reply:
I was also thinking about it, I'm not sure though. Adding it to git eliminates risk of using wrong version of the contract, but it unnecessarily makes project size bigger.

Reply:
In starknet.py we mostly keep only cairo source codes and not the compiled versions. We do have a logic for compiling contracts inside our package so it's not exactly 1:1 situation.